tests/e2e/tests/regression/insights/live-recommendations.e2e.ts

```diff
@@ -79,8 +77,8 @@ test
         // Verify that live recommendations displayed for each database separately
         // Verify that user can see the live recommendation "Update Redis database" when Redis database is less than 6.0 highlighted as RedisStack
         await t
-            .expect(await browserPage.InsightsPanel.getRecommendationByName(redisVersionRecom).exists).ok('Redis Version recommendation not displayed')
-            .expect(await browserPage.InsightsPanel.getRecommendationByName(redisTimeSeriesRecom).exists).notOk('Optimize Time Series recommendation displayed');
+            .expect(await browserPage.InsightsPanel.getRecommendationByName(redisVersionRecom).visible).ok('Redis Version recommendation not displayed')
+            .expect(await browserPage.InsightsPanel.getRecommendationByName(redisTimeSeriesRecom).visible).notOk('Optimize Time Series recommendation displayed');
```

Comment:
.visible + .notOk in TestCafe expect waits for ~30 seconds in most cases. 
So we used .exists to not so as not to increase the test time. Can you check it in this case?